.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout commit
+        uses: actions/checkout@v2
+
+      - name: Install correct version of Node
+        shell: bash -l {0}
+        run: nvm install
+
+      - name: Install correct version of Yarn
+        run: yarn policies set-version $(cat .yvmrc)
+
+      - name: Get Yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache Node modules
+        uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.OS }}-yarn-
```

Comment:
This is a little funky because the location of the Yarn cache varies between OS and which version of Yarn you're using. This is [the approach recommended](https://github.com/actions/cache/blob/master/examples.md#node---yarn) by the `cache` Action.